src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-501/rgw-501.component.html

```diff
@@ -0,0 +1,15 @@
+<nav aria-label="breadcrumb">
+  <ol class="breadcrumb">
+    <li i18n
+        class="breadcrumb-item">Object Gateway</li>
+  </ol>
+</nav>
+<alert type="info">
+  <strong>
+    <i class="fa fa-info-circle fa-fw alert-info"
+       aria-hidden="true"></i>
+  </strong>
+  <ng-container i18n>
+    The RGW service is not configured. Please consult the documentation about how to enable RGW for the dashboard.
```

Comment:
Shouldn't we add a link to the documentation?

Reply:
Makes sense, even though there is no Mimic-specific documentation at this point. For the time being, http://docs.ceph.com/docs/master/mgr/dashboard/#enabling-the-object-gateway-management-frontend would be the most accurate link.

However, http://docs.ceph.com/docs/mimic/mgr/dashboard/#enabling-the-object-gateway-management-frontend would be more appropriate, just in case the configuration and documentation evolve. I suggest to already use the latter, just to avoid we forget updating it :)